From d7820f6358c99b9b74b429b558dedc554f6da3b3 Mon Sep 17 00:00:00 2001
From: Mathew McBride <matt@traverse.com.au>
Date: Thu, 6 Aug 2026 17:37:45 +1000
Subject: [PATCH] firewall: fix typo in RDKB-21814 comment
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Identified by Copilot AI:
Typo in the comment: "managment" → "management".

Link: https://github.com/rdkcentral/utopia/pull/397#pullrequestreview-4872124064

---
 source/firewall/firewall.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/firewall/firewall.c b/source/firewall/firewall.c
index ff83557..f9795f6 100644
--- a/source/firewall/firewall.c
+++ b/source/firewall/firewall.c
@@ -6894,7 +6894,7 @@ int do_remote_access_control(FILE *nat_fp, FILE *filter_fp, int family)
 
 #if defined(_COSA_BCM_ARM_) || defined(_PLATFORM_RASPBERRYPI_) || defined(_PLATFORM_TURRIS_) || defined(_PLATFORM_BANANAPI_R4_) || defined(_PLATFORM_GENERICARM_)
     // RDKB-21814 
-    // Drop only remote managment port(8080,8181) in bridge_mode 
+    // Drop only remote management port(8080,8181) in bridge_mode 
     // because port 80, 443 will be used to access MSO page / local admin page.
 
     rc = syscfg_get(NULL, "mgmt_wan_httpaccess", query, sizeof(query));
